OCA-UFCG/oca-frontend
# Refactoring publications section

## Description

This PR updates the styling of the publications section

## How to test it

You'll only need to access the main page ("/") and go to the publications sections and check the updated version
